.github/CONTRIBUTING.md

```diff
@@ -9,11 +9,11 @@ Proper formatting
 Before making a pull request, please make sure your code is properly formatted
 by running the prepare commit script **before** issuing `git commit`.
 
-    ./scripts/prepare-commit.sh
+    ./scripts/prepare_commit.sh
 
 This can be automated by setting up the pre-commit hook properly.
 
-    ln -s ../../scripts/prepare-commit.sh .git/hooks/pre-commit
+    ln -s ../../scripts/prepare_commit.sh .git/hooks/pre-commit
```

Comment:
This change will make it hard to work on different branches. I would be in favor of leaving a sym link of prepare-commit.sh in place for a transition period at least (i.e. until 3.10 is eol).

Reply:
Good point, I've added the link